### PR TITLE
Recipe remove NPE

### PR DESCRIPTION
**Pull request by ghostflyby**
There is such output stack whose getItem returns null

### CODE REVIEW

File: src/main/java/com/dreammaster/recipes/RecipeRemover.java [hunk 1]
```diff
@@ -53,7 +53,13 @@ private static void stopBuffering() {
         int i = tList.size();
         tList.removeIf(r -> {
             ItemStack rCopy = r.getRecipeOutput();
-            if (rCopy == null) return false; // ????????????????????
+            if (rCopy == null) {
+                return false;
+            }
+            if (rCopy.getItem() == null) {
+                System.out.println("Someone is adding recipes with null items!");
```
Comment (eigenraven) [inline src/main/java/com/dreammaster/recipes/RecipeRemover.java:60]: Use the logger like below, not system.out:
```suggestion
                MainRegistry.Logger.warn("Someone is adding recipes with null items!");
```

Comment (boubou19) [reply src/main/java/com/dreammaster/recipes/RecipeRemover.java:60]: might also be a good idea to throw a stacktrace in the log to see where the recipe is

Comment (ghostflyby) [reply src/main/java/com/dreammaster/recipes/RecipeRemover.java]: Should I log it in warn (or other log level) or directly `Thread.dumpStack()`?

Comment (eigenraven) [reply src/main/java/com/dreammaster/recipes/RecipeRemover.java]: Log it in warn

**Review by boubou19 — APPROVED** [review]

**Pull request merged**